src/retsnoop.c

```diff
@@ -411,7 +411,7 @@ int main(int argc, char **argv, char **envp)
 	skel->rodata->use_kprobes = env.attach_mode != ATTACH_FENTRY;
 	memset(skel->rodata->spaces, ' ', sizeof(skel->rodata->spaces) - 1);
 
-#ifdef __x86_64__
+#if defined(__x86_64__) || defined(__aarch64__)
```

Comment:
we should probably have a properly named #define for this at this point, something like:

```
#if defined(__x86_64__) || defined(__aarch64__)
#define RETSNOOP_SUPPORTS_ARG_CAPTURE 1
#endif
```

and use `#ifdef RETSNOOP_SUPPORTS_ARG_CAPTURE` in all relevant places?